packages/common/core-utils/src/assert.ts

```diff
@@ -18,7 +18,9 @@
 export function assert(condition: boolean, message: string | number): asserts condition {
 	if (!condition) {
 		throw new Error(
-			typeof message === "number" ? `0x${message.toString(16).padStart(3, "0")}` : message,
+			typeof message === "number"
+				? `FF Assert: 0x${message.toString(16).padStart(3, "0")}`
+				: `FF Assert: ${message}`,
```

Comment:
i don't love the FF prefix, but i understand it, did you have any other ideas?

Reply:
We should agree on the problems being solved. The monitor is fixed without this by inspecting the callstack. So this is now something to consider for ourselves or partners who look closely at our logs and errors.

I like the message prefix because it's in your face. But I don't feel very strongly. What do you think?

Reply:
I don't think I was clear, it the string "FF Asssert" as a prefix i think could be improved. i don't mind a prefix in general

Reply:
@alexvy86 discussion is good, and since we aren't worried about it for monitors, i'm not too concerned about a potential conflict, just make sure none of the standard env assert functions do the same prefix and i'd say we're good.

Reply:
I'd go with "Fluid Framework Internal Error: ". That provides enough information that someone equipped with a web search engine could find where to report the issue and know the issue is a fluid bug, not invalid use of fluid, even if they are not a direct consumer of Fluid Framework (for example a customer of an app that prints exception, or a user of an SDK that uses Fluid as an implementation detail)

Reply:
[A bystander question about the suggestions above:] 
If a customer builds their own driver layer, and use the assert from core-utils package, would this new driver also have the potential to report "Fluid Framework Internal Error"? If yes, does the prefix still seem correct?

Reply:
While it feels verbose, Craig's reasoning is very sound to me.

To Pragya's question, would we expect driver developers to use our assert function? My understanding is that it exists solely for our team/Microsoft partners who we debug with. In that light I wouldn't expect drivers to use our function.

Reply:
our assert function is under @fluidframework/core-utils/internal so it should not be used by anything not part of Fluid Framework.

Reply:
Correction. its under legacy+alpha. It should end up in internal at some point though.